Pipfile

```diff
@@ -8,6 +8,7 @@ markdown-include = "==0.5.1"
 mdx-truly-sane-lists = "==1.0"
 mkdocs = "==0.15.3"
 pymdown-extensions = "==1.1"
+mdx_truly_sane_lists = "==1.1.1"
```

Comment:
It's so weird that I installed `mdx-truly-sane-lists` before (3 lines above). I don't know what the correct one is.

Reply:
Did you add this by hand? Because running `pipenv install mdx_truly_sane_lists` reports: `Requirement already satisfied: mdx_truly_sane_lists`. What was the error you received?

Reply:
Oh yah I added this by hand, nice catch, will close PR 👍